File: src/questions/parkinglot/java/strategy/fee/FlatRateFeeStrategy.java
package questions.parkinglot.java.strategy.fee;

import questions.parkinglot.java.entity.ParkingTicket;

public class FlatRateFeeStrategy implements FeeStrategy {

    private final double flatRate;

    public FlatRateFeeStrategy(double flatRate) {
        this.flatRate = flatRate;
    }

    @Override
    public double calculateFee(ParkingTicket ticket) {
        return flatRate;
    }
}

File: src/questions/parkinglot/py/strategy/fee/FlatRateFeeStrategy.py
from questions.py.entity import ParkingTicket
from questions.py.strategy.fee.FeeStrategy import FeeStrategy


class FlatRateFeeStrategy(FeeStrategy):
    def __init__(self, flat_rate: float):
        self._flat_rate = flat_rate

    def calculate_fee(self, ticket: ParkingTicket) -> float:
        return self._flat_rate
